packages/jest-transform/src/ScriptTransformer.ts

```diff
@@ -343,19 +344,26 @@ export default class ScriptTransformer {
           instrument,
         );
 
-        wrappedCode = wrap(transformedSource.code, ...extraGlobals);
+        const wrapResult = wrap(transformedSource.code, ...extraGlobals);
```

Comment:
I'm not too sure about this. Maybe `v8-to-istanbul` could offload more source map work to `istanbul-lib-source-maps` to make it closer to how just getting `global.__coverage__` works today?

/cc @bcoe

Reply:
It's not currently my intention to move towards using `istanbul-lib-source-maps` for `v8-to-istanbul`. As with the `istanbul-api` module, I think this adds an unnecessary abstraction between the modules that are actually doing the heavy lifting:

* [source-map](https://www.npmjs.com/package/source-map).
* [convert-source-map](https://www.npmjs.com/package/convert-source-map)

The other technical reason, is that `v8-to-istanbul` is using the newest version of `source-map` (which is async) and I don't believe we were ever able to upload to the newest version of `source-map` with Istanbul, because too much of the dependency chain relies on synchronous behavior.

Reply:
If I were you, I'd probably build a thin wrapper layer in Jest, if you want to expose a similar API for both.